scripts/run_lighthouse_tests_test.py

```
@@ -487,7 +484,7 @@ def mock_popen(
         swap_emulator_mode = self.swap(constants, 'EMULATOR_MODE', False)
 
         with swap_popen, swap_isdir, swap_build:
-            with self.swap_elasticsearch_dev_server, self.swap_dev_appserver:
+            with self.swap_dev_appserver:
```

Ruff (0.16.1)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)